tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -19,6 +19,8 @@ fn storage_key(path: &str) -> String {
 }
 
 #[test]
+#[ignore]
+// this test is outdated, so it is ignored
```

Comment:
Yep (also badly named...). There was this PR to clean it up (https://github.com/anoma/anoma/pull/1229) but that's a bit stale now probably. This test is still testing something we should continue testing (that a random actor cannot write arbitarily to `EthBridge` VP storage) but this test needs cleaning up.